app.js

```diff
@@ -9,85 +9,134 @@ const weather = {};
+const day1IconElement = document.querySelector(".day-weather-icon1");
+const day2IconElement = document.querySelector(".day-weather-icon2");
+const day3IconElement = document.querySelector(".day-weather-icon3");
+const day1TemperatureElement = document.querySelector(".day-temperature-value1");
+const day2TemperatureElement = document.querySelector(".day-temperature-value2");
+const day3TemperatureElement = document.querySelector(".day-temperature-value3");
+const day1DescriptionElement = document.querySelector(".day-weather-description1");
+const day2DescriptionElement = document.querySelector(".day-weather-description2");
+const day3DescriptionElement = document.querySelector(".day-weather-description3");
+/////////////////////////////////////////////////////////////////////////////////////////////////
+
 // CHECK IF BROWSER SUPPORTS GEOLOCATION
 if('geolocation' in navigator){
     navigator.geolocation.getCurrentPosition(setPosition, showError);
 }else{
     notificationElement.style.display = "block";
     notificationElement.innerHTML = "<p>Browser Not Supporting Geolocation</p>"
-}
+};
 
 // SHOW ERROR WHEN THERE IS AN ISSUE WITH GEOLOCATION SERVICE
 function showError(error){
     notificationElement.style.display = "block";
-    notificationElement.innerHTML = `<p>${error.message}</p></p>`;
-}
+    notificationElement.innerHTML = `<p>${error.message}</p>`;
+};
 
 // SET USER'S POSITION
 function setPosition(position){
     let latitude = position.coords.latitude;
     let longitude = position.coords.longitude;
 
-    getWeather (latitude, longitude);
-}
-
-// GET WEATHER FROM API
-function getWeather(latitude, longitude){
-    let api = `https://api.openweathermap.org/data/2.5/weather?lat=${latitude}&lon=${longitude}&appid=${key}`;
+    currentGetWeather (latitude, longitude);
+    dayGetWeather (latitude, longitude);
+};
 
-    // OTHER DATA THAT CAN BE FETCHED: COUNTRY, TIMEZONE, PRESSURE, HUMIDITY, WIND SPEED, WIND DIRECTION, FEELS LIKE, MAX/MIN TEMP, SUNRISE/SUNSET
+// GET CURRENT WEATHER FROM API
+function currentGetWeather(latitude, longitude){
+    let currentAPI = `https://api.openweathermap.org/data/2.5/weather?lat=${latitude}&lon=${longitude}&appid=${key}`;
 
-    fetch(api)
+    fetch(currentAPI)
         .then(function(response){
-            let data = response.json();
-            return data;
+            return response.json();
         })
         .then(function(data){
-            weather.temperature.value = Math.floor(((data.main.temp - KELVIN) * (9/5)) + 32); //Converts from Kelvin to Fahrenheit
-            weather.description = data.weather[0].description; //MAIN INSTEAD OF DESCRIPTION?
-            weather.iconId = data.weather[0].icon;
-            weather.city = data.name;
-            weather.country = data.sys.country;
+            weather.temperature.currentValue = Math.floor(((data.main.temp - KELVIN) * (9/5)) + 32); //Converts from Kelvin to Fahrenheit
```

Comment:
You should move all of this logic into the above then handler (ideally into a separate function and just call the functions from the handler). Doesn't look like you're doing any asynchronous operations after calling the api so no need to keep calling then handler which return promises.

Reply:
Ahhhh light bulb moment. I see what you're saying. When I finished re-editing this, I should just commit changes and then open another PR?

Reply:
Yeah Just make a new commit to the PR.